src/index.js

```diff
@@ -1,6 +1,17 @@
+/* eslint-disable no-undef */
```

Comment:
Se for problema de Globals é possivel atualizar no .eslint, se for outra questão precisamos ver.

Reply:
sim, é por conta uma variável global, a document. Concordo em atualizar no .eslint mas provavelmente com o código completo, nem vou usá-la.